railties/lib/rails.rb

```diff
@@ -45,6 +45,7 @@ def application
     end
 
     delegate :initialize!, :initialized?, to: :application
+    delegate :credentials, to: :application
```

Comment:
This is just me, but I would almost prefer a normal method like `configuration` here so it can be properly documented.

I'm not sure this change will be accepted, however.

Reply:
Thanks for the feedback. I went back and forth on the implementation. As far as your comment about not thinking it will be accepted... are you referring to the change in its current state or at all? If not at all, would you explain your feeling on that?

Reply:
Mainly, getting approval on the new class method. Maybe asking on [discord](https://rubyonrails.org/2022/6/13/rails-discord-server-is-now-open-to-the-public) would be a good place to start!